### ISSUE TITLE
No results obtained using default parameters

### ISSUE THREAD

**Issue by jce2090** [TreezzZ/DHN_PyTorch#2]
Hello, I want to use DHN to verify palmprint recognition code. For this I use default parameters to train the DHN network, my settings are as follows:
parser = argparse.ArgumentParser(description='DHN_PyTorch')
    parser.add_argument('--dataset', default='cifar-10',
                        help='Dataset name.')
    parser.add_argument('--root', default='../cifar-10',
                        help='Path of dataset')
    parser.add_argument('--code-length', type=int, default=128, 
                        help='Binary hash code length.')
    parser.add_argument('--arch', default='alexnet', type=str,
                        help='CNN model name.(default: alexnet)')
    parser.add_argument('--batch-size', default=256, type=int,
                        help='Batch size.(default: 256)')
    parser.add_argument('--lr', default=1e-2, type=float,
                        help='Learning rate.(default: 1e-5)')
    parser.add_argument('--max-iter', default=500, type=int,
                        help='Number of iterations.(default: 500)')
    parser.add_argument('--num-workers', default=6, type=int,
                        help='Number of loading data threads.(default: 6)')
    parser.add_argument('--topk', default=-1, type=int,
                        help='Calculate map of top k.(default: all)')
    parser.add_argument('--gpu', default=True, type=int,
                        help='Using gpu.(default: False)')
    parser.add_argument('--lamda', default=1, type=float,
                        help='Hyper-parameter.(default: 1)')
    parser.add_argument('--seed', default=3367, type=int,
                        help='Random seed.(default: 3367)')
    parser.add_argument('--evaluate-interval', default=10, type=int,
                        help='Evaluation interval.(default: 10)')

But using this set of parameters, I can't get the result of MAP, and LOSS hasn't dropped. Is there any error in my verification method?

**Comment by TreezzZ** [comment]
Maybe I forget to modify the default parameters in configuration, you can try to set different learning rate and lamda to train the network.
